opentelemetry.html.md.erb

```diff
@@ -4,35 +4,36 @@ owner: Logging and Metrics
 ---
 
 You can deploy an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) for <%= vars.app_runtime_abbr %>
-to egress metrics.
+to egress traces, metrics, and logs.
 
-When the OpenTelemetry (OTel) Collector is deployed, the Loggregator Forwarder Agent forwards platform and application metrics
-to the OTel Collector, which then egresses metrics using one or more metric exporters that you configure.
+When the OpenTelemetry (OTel) Collector is deployed, the Loggregator Forwarder Agent forwards platform and application metrics to the OTel Collector, which then processes traces, metrics, and logs using one or more pipelines that you configure.
 
-<p class="note caution">
-<span class="note__title"><strong>Caution</strong></span>
-OpenTelemetry Collector is currently an experimental feature and should not be used in production systems.</p>
+# <a id='collector-config'></a> Collector configuration structure
 
-## <a id='metric-exporter-config'></a> Metric Exporter configuration
 
-The OTel Collector has a standard YAML-based file format for configuration. When deploying <%= vars.app_runtime_abbr %>
-with Otel Collector support, provide only the exporter configuration.
+The OTel Collector has a standard YAML-based file format for configuration. 
 
-<p class="note important">
-<span class="note__title"><strong>Important</strong></span>
-Specify only the configuration under the <code>exporters</code> section of the OpenTelemetry Collector configuration. It
-is not currently possible to configure Receivers or Processors.</p>
+The structure of <%= vars.app_runtime_abbr %> configuration file consists of three classes of pipeline components that access telemetry data:
+
+* [Exporters](#exporter-config)
```

Comment:
Might want to keep this as `#metric-exporter-config` based on @pspinrad's [previous comment](https://github.com/cloudfoundry/docs-loggregator/pull/92#discussion_r1752946093). Unless we want to go change the partial as well 🤷